src/harness/reference_models/interference/interference.py

```diff
@@ -15,21 +15,16 @@
 ==================================================================================
   Compute Interference caused by a grant for all the incumbent types
   APIs in this file are used by IAP and Aggregate Interference Reference Models
-
```

Comment:
Why are you removing in this file all empty lines which improves readability ?

Reply:
Added   back the empty lines